docs/en/maintain/backup.md

```diff
@@ -0,0 +1,202 @@
+# Backup and restore
+
+## High Availability Description
+
+* The nameserver cannot be fully hung up. Failover and recovery cannot be done if all hangs up
+* High availability is not guaranteed in the case of a single copy
+* In the case of two replicas and two nodes, the tablet can only hang on one node at most
+* In the case of two replicas and multiple nodes, the two tablet nodes where the same shard is located cannot hang at the same time
+* In the case of three replicas and three nodes, hanging a tablet node can automatically perform failover and data recovery. Hanging two tablet nodes does not guarantee automatic failover and data recovery, but reads and writes can return to normal within minutes
```

Comment:
hanging a tablet => a tablet failure
Hanging two tablet nodes => Failure of two tablet nodes